tests/unit/test_channels/test_backends.py

```diff
@@ -77,10 +77,11 @@ async def test_pub_sub_unsubscribe(channels_backend: ChannelsBackend) -> None:
     await channels_backend.publish(b"something", ["foo"])
 
     event_generator = channels_backend.stream_events()
+    await asyncio.wait_for(async_next(event_generator), timeout=0.01)
     await channels_backend.unsubscribe(["foo"])
-    await channels_backend.publish(b"something", ["bar"])
+    await channels_backend.publish(b"something else", ["bar"])
 
-    assert await asyncio.wait_for(async_next(event_generator), timeout=0.01) == ("bar", b"something")
+    assert await asyncio.wait_for(async_next(event_generator), timeout=0.01) == ("bar", b"something else")
```

Comment:
I don't think this test is correct anymore.

It's supposed to show that unsubscribing works, but now it has consumed all events on the `foo` channel *before* unsubscribing, so even if unsubscribing does not work, the test would still be green.

Reply:
For some reason on the `psycopg` backend, it was returning (`foo`, `something`) instead of `bar`